sample_global_configs.json

```diff
@@ -44,29 +43,10 @@
     },
 
     "TABULAR": {
-        "DEMOGRAPHICS": {
-            "SEX": {
-                "VERSION": "",
-                "FILENAME": ""
-            },
-            "DATE_OF_BIRTH": {
-                "VERSION": "",
-                "FILENAME": ""
-            }
-        },
-        "ASSESSMENTS": {
-            "UPDRS_3": {
-                "VERSION": "",
-                "FILENAME": ""
-            },
-            "MOCA": {
-                "VERSION": "",
-                "FILENAME": ""
-            },
-            "GROUP": {
-                "VERSION": "",
-                "FILENAME": ""
-            }
-        }
+        "data_dictionary": {
+            "PATH": "",
+            "VERSION": "",
+            "URL": ""
```

Comment:
What is URL for?

Reply:
Sometimes it could be a download link on the data publishing website.